.travis.yml

```diff
@@ -5,9 +5,13 @@ env:
 - TOX_ENV=py27
 - TOX_ENV=py33
 - TOX_ENV=py34
+- TOX_ENV=py27-build-theme-mkdocs
+- TOX_ENV=py34-build-theme-mkdocs
+- TOX_ENV=py27-build-theme-readthedocs
+- TOX_ENV=py34-build-theme-readthedocs
```

Comment:
I actually would prefer to keep them, I was trying to decide which made most sense to keep - but I think it should be all or none.

It would be nicer to bundle them together as one integration test run, but I can't see how to do that.